README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
@hferentschik @LalatenduMohanty Yes, it seems that the released proxy feature is not working due to passing wrong environment variables `HTTP_PROXY` etc instead of `PROXY` to sccli.

IMO, it is ok to release this without any depreciation since most of the users are using shipped Vagrantfiles which has properly set proxy configurations.
Otherwise we would have got at least one mail of proxy not being setup properly if they were using 
service-manager way.

WDYT ?